test/functional/update/update-slow-server.bats

```diff
@@ -40,6 +40,7 @@ test_teardown() {
 		Downloading packs...
 		Curl: Partial file downloaded from '.*pack-test-bundle-from-10.tar'
 		Starting download retry #1 for .*100/pack-test-bundle-from-10.tar
```

Comment:
I know this is not part of this review, but we should consider adding the ' ' around the url from the message that goes like "Starting download retry ....", all other messages use it, so it would look more consistent.